Unity/NuGet/VedalAI.NeuroSdk.Unity.targets

```diff
@@ -0,0 +1,14 @@
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <Target Name="PreRestoreEnsureUnityReference" BeforeTargets="CoreCompile;Restore;CollectPackagesForRestore">
+        <PropertyGroup>
+            <_HasUnityReference Condition="'%(PackageReference.Identity)' == 'UnityEngine.Modules'">true</_HasUnityReference>
```

Comment:
Someone might be directly referencing the unity dlls. Should we take that into account instead of forcing them to use UnityEngine.Modules?

Reply:
I don't think there's much point in chasing down every single assembly this code depends on - I just tried and it did not feel fun or worth the effort - so I'd like to instead give the option to override the check.
```diff
         <Error Condition="'$(_HasUnityReference)' != 'true'"
-                Text="This SDK depends on Unity. Please add a PackageReference to any version of Unity to your project." />
+                Text="This SDK depends on Unity. Please add a PackageReference to any version of UnityEngine.Modules to your project.
+ To override this check (e.g. if you are referencing Unity dlls directly from disk), set the _HasUnityReference property to true in a PropertyGroup in your project." />
```
How about this?

Reply:
👍